src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+/* general styles */
+:root {
+  --main-color: #00acdc;
+}
+
+* {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  margin: 0;
+  padding: 0;
```

Comment:
Don't use for `*` selector any rules instead of `box-sizing`.
Move `font-family` and `font-size` to the `body` selector.
And `margin`/`padding` to the blocks which need these rules.